dist/stampit.js

```diff
@@ -710,9 +738,13 @@ isStamp = function isStamp(obj) {
  * @return {Function}             A composable stampit factory
  *                                (aka stamp).
  */
-convertConstructor = function convertConstructor(Constructor) {
-  return stampit().methods(Constructor.prototype).enclose(Constructor);
-};
+function convertConstructor(Constructor) {
+  var stamp = stampit();
+  mixInChainFunctions(stamp.fixed.methods, Constructor.prototype);
+  stamp.fixed.state = mergeChain(stamp.fixed.state, Constructor.prototype);
```

Comment:
You are looking at the wrong commit. Please, check the combined changes. Here is the link to the one:
https://github.com/ericelliott/stampit/pull/48/files#diff-f547c18e2953252cb96744460b29a7cfR205